src/cheri/insns/jalr_32bit.adoc

```diff
@@ -63,10 +63,11 @@ again the motivation is CHERIoT, which uses register selectors to distinguish be
 
 [NOTE]
 =====
-When a sealed capability is passed as the input to <<JALR_CHERI>>, its address must have bit zero clear and the instruction must have a zero offset, or the target <<pcc>> will have its {ctag} set to zero, since updates to its address are interpreted with <<SCADDR>> semantics.
+When a sealed capability (i.e. a sentry) is passed as the input to <<JALR_CHERI>>, its address must have bit zero clear and the instruction must have a zero offset, or the target <<pcc>> will have its {ctag} set to zero, since updates to its address are interpreted with <<SCADDR>> semantics.
+Sentries are used as secure function pointers, and so cannot be offset on dereference.
```

Comment:
This added line feels more appropriate for a programmer's guide.  Architecturally, the inability to offset a sentry is simply a consequence of sentries' sealed nature.